File: multiplesofnum.py
#!/usr/bin/python

import sys

# check that a filename argument was provided, otherwise
if len(sys.argv) < 2:
  raise Exception("Filename must be first argument provided")
  
filename = sys.argv[1]
lines = []

# open file in read mode, assuming file  is in same directory as script
try:
  file = open(filename, 'r')

  # read Fibbonacci indexes from file into list
  lines = file.readlines()
  file.close()
except IOError:
  print "File '%s' was not found in current directory" % filename
  
lines = [line.replace('\n', '') for line in lines] 

for line in lines:
  digits = line.split(",")
  floor = int(digits[0])
  base = int(digits[1])
  
  multiple = 1
  
  while (multiple * base) < floor:
    multiple += 1
  
  print multiple * base

File: multiplesofnum.rb
#!/usr/bin/ruby

filename = ARGV[0]

if filename.nil?
  exit
end

file = File.new(filename)

while (line = file.gets)
  digits = line.split(",")
  floor = digits[0].to_i
  base = digits[1].to_i
  multiple = 1
  value = base*multiple
  
  while value < floor
    multiple += 1
    value = base*multiple
  end
  
  puts value
end